cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -336,6 +342,7 @@ def test_inventory_refresh_westindia_azure():
     pass
 
 
+@test_requirements.relationships
```

Comment:
I think this file contains only `relationships` tests then why not we go with `global` marker.

Reply:
There are some test cases which are assigned to ```anikifor```. Although ```relationships``` only deals with provider's details page. Hence I did not go with global marker.